ethereum/core/src/test-support/java/tech/pegasys/pantheon/ethereum/core/ExecutionContextTestFixture.java

```diff
@@ -55,6 +55,7 @@ private ExecutionContextTestFixture(
             new NoOpMetricsSystem());
     this.stateArchive =
         new WorldStateArchive(new KeyValueStorageWorldStateStorage(keyValueStorage));
+    genesisState.writeStateTo(stateArchive.getMutable());
```

Comment:
There's another call to `writeStateTo` a few lines down - looks like one of these should be deleted.

Reply:
So there is. Fixed.